agentstack/cli/cli.py

```diff
@@ -16,17 +18,67 @@
 from ..utils import open_json_file, term_color, is_snake_case
 
 
-def init_project_builder(slug_name: Optional[str] = None, skip_wizard: bool = False):
+def init_project_builder(slug_name: Optional[str] = None, template: Optional[str] = None, use_wizard: bool = False):
     if slug_name and not is_snake_case(slug_name):
         print(term_color("Project name must be snake case", 'red'))
         return
 
-    if skip_wizard:
+    if template is not None and use_wizard:
+        print(term_color("Template and wizard flags cannot be used together", 'red'))
+        return
+
+    template_data = None
+    if template is not None:
+        url_start = "https://"
+        if template[:len(url_start)] == url_start:
+            # template is a url
+            response = requests.get(template)
+            if response.status_code == 200:
+                template_data = response.json()
+            else:
+                print(term_color(f"Failed to fetch template data from {template}. Status code: {response.status_code}", 'red'))
+                sys.exit(1)
+        else:
+            with importlib.resources.path('agentstack.templates.proj_templates', f'{template}.json') as template_path:
```

Comment:
I just added it, but `utils.get_package_path` seems to make these interactions a little more robust, especially across versions and avoids the `with` statement.

Reply:
i saw that in the PR! once that's in main, ill use it here instead